packages/flutter/lib/src/widgets/table.dart

```diff
@@ -81,6 +81,70 @@ class _TableElementRow {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=_lbE0wsVZSw}
 ///
+/// {@tool dartpad --template=stateless_widget_material}
```

Comment:
I recommend using `--template=stateless_widget_scaffold` instead - it includes an AppBar and a white background.

Reply:
Changed in https://github.com/flutter/flutter/pull/69588/commits/1678ebfa303062e075032c62c727d19c036998ed